lib/collection/variable-list.js

```diff
@@ -194,8 +184,8 @@ _.assign(VariableList.prototype, /** @lends VariableList.prototype */ {
         }
 
         // if we request a particular environment, we get the one, else we convert all environments to JSON
-        return arguments.length ? (env && toJSON(env)) : _.map(this._environments, function (environment) {
-            return toJSON(environment);
+        return arguments.length ? (env && PropertyBase.toJSON(env)) : _.map(this._environments, function (environment) {
```

Comment:
The function in `_.map` can be replaced by `PropertyBase.toJSON.bind(PropertyBase)`.